doc/man7/EVP_KEM-MLKEM.pod

```diff
@@ -0,0 +1,64 @@
+=pod
+
+=head1 NAME
+
+EVP_KEM-MLKEM,
+EVP_KEM-ML-KEM
```

Comment:
IMO, this should look like this:
```suggestion
EVP_KEM-ML-KEM-768
```

The name after the `EVP_KEM-` prefix should be the name as it is fetched from the provider, i.e. the names as listed in the `PROV_NAMES_MLKEM768` define. It is not possible to "fetch" an algorithm called "ML-KEM" or "MLKEM" so we should not list them here. We can extend the list to include "ML-KEM-512" and "ML-KEM-768" as we implement them

Reply:
ACK. See below. Let me play with the doc-nits script how to bring your logical argument (that I agree with) and that check in line...

Reply:
Now changed adding all possibly key types -- but had to leave "generic" EVP_KEM-MLKEM to satisfy `make doc-nits`.